packages/integration-tests/index.js

```diff
@@ -539,7 +539,7 @@ async function addRulesWithPostfix(config, dataDirectory, overrides, postfix) {
 
       rule = Object.assign(rule, overrides);
       const ruleTemplate = Handlebars.compile(JSON.stringify(rule));
-      const templatedRule = JSON.parse(ruleTemplate(config));
+      const templatedRule = JSON.parse(ruleTemplate(Object.assign({}, config, process.env)));
```

Comment:
I'm not sure I understand why we are merging `process.env` onto this object?

Reply:
There are some places in the specs where it relied on `config.awsAccountId` and `config.awsRegion` being defined in the test configs, especially template files. It didn't really make sense to recreate defining and committing those as they are in `app/config.yml` when they are available in the environment in CI, so instead they were moved into the `example/.env` file, but the test helpers and handlebars still expect them to be in that config object.

Reply:
b70dd0051 👍